Source/RunActivity/Viewer3D/Scenery.cs

```diff
@@ -480,7 +480,28 @@ public WorldFile(Viewer viewer, int tileX, int tileZ, bool visible)
                     }
                     else if (worldObject.GetType() == typeof(StaticObj))
                     {
-                        if (isAnalogClock)
+                        isAnimatedClock = false;                                                              //Preset
+                        //          preTestShape for lookup if it is an animated clock shape with subobjects named as clock hands 
+                        StaticShape preTestShape = (new StaticShape(viewer, shapeFilePath, worldMatrix, shadowCaster ? ShapeFlags.ShadowCaster : ShapeFlags.None));
+                        if (preTestShape.SharedShape.Animations != null)                                      // shape has an Animation at all
+                        {
+                            if (preTestShape.SharedShape.Animations[0].anim_nodes.Count > 1)                  // shape has more than 1 anim node
+                            {
+                                //lookup in all anim nodes of the shape for subobjects named as clock hands
+                                foreach (var animNodes in preTestShape.SharedShape.Animations[0].anim_nodes)
+                                {
+                                    if (animNodes.Name.ToLowerInvariant().IndexOf("hand_clock") == 6)        //Shape anim node name contains "hand_clock"
```

Comment:
This checks that the string "hand_clock" is in the sixth position, not just that it exists. I think you meant to use .Contains("hand_clock") here.